### DISCUSSION TITLE
Allow to add Mastodon profile URLs

### DISCUSSION THREAD

**Discussion by j9t** [forem/forem#18913] [Feature Requests]
Similar to featuring GitHub and Twitter on DEV profiles, it would be great if Mastodon profiles could be added as well.

This seems useful not only in the light of [Mastodon’s growing adoption](https://trends.builtwith.com/cms/Mastodon), but because parts of the community have noticeably and what feels like largely moved away from Twitter, to Mastodon (take e.g. instances like [front-end.social](https://front-end.social/)). Allowing to add Mastodon profile URLs could therefore support connecting within the community.

**Comment by yheuhtozr** [comment]
Much nicer if support could be extended to any Fediverse software, not specifically Mastodon (only the issue of icon choice if fortunate?).

**Comment by erinb223** [comment]
Adding [this comment](https://dev.to/andypiper/comment/2679p) from @andypiper, as I think it presents an exciting perk to having a DEV profile (quick verification on Mastodon).

> Now, add link rel="me" to DEV profile pages, and we can verify ourselves on Mastodon!

**Reply by GeorgeWL** [reply]
Also the great thing with `rel=me`works for lots of other services too

**Comment by heliocola** [comment]
Hi there, I just want to bump this conversation up.
I just noticed I can't link my Fediverse profile, like I can in several other apps.

I am thinking that could be a section under Account Settings that includes social profiles and online presence.
I think RailsDevs.com did something quite interesting with specific icons and URLs, but for Dev.To may have a Generic section where the user add the label and the value, thus allowing the user the ability to promote the name of their Fediverse instance.

That can later be extended to allowing the user to upload an SVG that better represent their Fediverse instance.

Anyway, what do y'all think?
